app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::Base
+    before_action :configure_permitted_parameters, if: :devise_controller?
+  
+    protected
+  
+    def configure_permitted_parameters
+        def configure_permitted_parameters
+            devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password, :kind) }
+        end
+    end
```

Comment:
Nested methods?

Reply:
It was done by mistake. Let me correct it.